WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReadersV2/CardReaderSettingsConnectedViewController.swift

```diff
@@ -127,9 +127,10 @@ private extension CardReaderSettingsConnectedViewController {
             }
         } else if let readerUpdateProgress = viewModel.readerUpdateProgress {
             // If we are updating a reader, show the progress alert
-            settingsAlerts.updateProgress(from: self, requiredUpdate: false, progress: readerUpdateProgress, cancel: { [weak self] in
-                self?.viewModel?.cancelCardReaderUpdate()
-            })
+            settingsAlerts.updateProgress(from: self,
+                                          requiredUpdate: false,
+                                          progress: readerUpdateProgress,
+                                          cancel: viewModel.cancelCardReaderUpdate)
```

Comment:
nit: it took me some time to realize why this change hides the cancel CTA (by whether `cancel` closure is nil or not), would it be more clear if there is another boolean from the view model like `isCancelButtonShown`/`showsCancelButton` based on the progress? This way, the cancel closure can stay the same and we just change the cancel button visibility boolean.